.github/workflows/ci.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
       - name: Install Dependencies
         if: steps.mix-cache.outputs.cache-hit != 'true'
+        working-directory: elixir
```

Comment:
How was this working previously without this nor a `cd elixir` ?

Reply:
Because some old cache was stored. Just changing any elixir dependency breaks the execution